### PR TITLE
Improve onboarding

### PR DESCRIPTION
**Pull request by rickycodes**
**Description**

Changes to improve on-boarding for the next human that comes along

**List of items to address:**

- env files
  - [x] add missing example files (I think android?)
  - [x] add check for env files in `build.sh`
- Sdk requirements for Android
  - `sdk.dir` needs to be defined in `local.properties` and/or `$ANDROID_HOME` needs to be defined in `$PATH` (`~/.profile`, or `~/.bash_profile` etc.)
- `secret-tool` is a requirement
  - <strike>[ ] test for this eg: `which secret-tool` in the `build.sh`</strike>
- licence needs to be accepted, this is easiest to accomplish with android studio
- [x] document all of the above in the `README.md`
- <strike>[ ] step through changes in a VM to verify</strike>

### CODE REVIEW

**Review by brunobar79 — APPROVED** [review]
Looks good to me. Thanks for doing this Ricky!

Can you add one more note in the read me mentioning that you need to rename the .env.example files first?

After that I'm okay with merging it and see if the CI build passes when merged into develop.

The reason why it might not pass is because the CI machine is macOS based and I've problems with certain shell syntax before where some of it works in linux only or macOS only.

if it doesn't we can revert and fix.

File: README.md [hunk 1]
```diff
@@ -28,6 +28,8 @@ yarn install
 
 - Running the app on Android:
 
+You'll need the `secret-tool` (a part of [libsecret-tools](https://launchpad.net/ubuntu/bionic/+package/libsecret-tools) package on Debian/Ubuntu based distros) and `watchman` binaries on your machine. You'll also want to have the android SDK; the easiest way to do that is by installing [Android Studio](https://developer.android.com/studio). Additionally, you'll need to install the Google Play Liciencing Libray via the SDK Manager in Android Studio.
```
[the remark this answers is not in the document]
Comment (rickycodes) [reply README.md]: I also spelled "Library" incorrectly :woman_facepalming: 

should be good now

**Comment by rickycodes** [comment]
> Can you add one more note in the read me mentioning that you need to rename the .env.example files first?

done!

> The reason why it might not pass is because the CI machine is macOS based and I've problems with certain shell syntax before where some of it works in linux only or macOS only.

Usually I try to use POSIX sh syntax which can be a little harder to work with, but in some linux there is no bourne again shell (or the version is older than what ships with OSX). For example I know there is a mismatch between OSX and Alpine linux (which a lot of build systems use because it's lightweight). However I think my changes should be okay in _this_ case.

File: android/app/fabric.properties [hunk 2]
```diff
@@ -1,3 +1,5 @@
+#AUTOGEN FABRIC PROPERTIES
+#Wed, 15 Jan 2020 16:20:22 -0500
 #Contains API Secret used to validate your application. Commit to internal source control; avoid making secret public.
 #Thu Aug 23 18:04:07 EDT 2018
-apiKey=0
+apiKey=9216d1fa651341733ac1220be0cb2e62d4f703a1
```
Comment (brunobar79) [inline android/app/fabric.properties]: You don't want to merge this

Comment (brunobar79) [reply android/app/fabric.properties]: Actually can you revert and force push to make it disappear?

**Pull request merged**